morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -109488,6 +109558,7 @@ La_o^di^ce_ni		<orth>Lau-di^ce_ni</orth>
 La_o^me^don		
 :le:Laomedon
 :no:La_o^me^d	on_ontis masc
+:no:La_u^me^d	on_ontis masc
```

Comment:
Same as above - can we or `orth` / `variant` this (or introduce a new term that is more appropriate)?

Reply:
added variant tag